aptos-move/e2e-move-tests/src/aggregator_v2.rs

```diff
@@ -90,83 +99,29 @@ impl ElementType {
 }
 
 #[derive(Debug)]
-pub struct AggLocation<'a> {
-    account: &'a Account,
+pub struct AggLocation {
+    address: AccountAddress,
     element_type: ElementType,
     use_type: UseType,
```

Comment:
I think it would be great to add a comment about why we need this (to identify in resource/group, etc.)